examples/with-apollo/src/Home.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import logo from './react.svg';
+import './Home.css';
+import { Link } from 'react-router-dom';
+
+class Home extends Component {
+  static async getInitialProps({ req, res, match, history, location, ...ctx }) {
+    return { stuff: 'whatevs' };
```

Comment:
Can we fetch some data from a public graphql server and render the results?

Reply:
Yes. I'm on it